sdk/v2/core/projects_test.go

```diff
@@ -23,6 +23,81 @@ func TestProjectListMarshalJSON(t *testing.T) {
 	metaTesting.RequireAPIVersionAndType(t, ProjectList{}, "ProjectList")
 }
 
+func TestValidateProjectID(t *testing.T) {
```

Comment:
Same as [this](https://github.com/brigadecore/brigade/pull/1437/files#r660822823)